src/main/java/bowlingexercise3/Bowling.java

```diff
@@ -34,6 +57,19 @@ public void turns(String total) {
 
     }
 
+    private void validateInput(String input) throws IllegalArgumentException {
+
+        //should have 10 frames
+        // List<String> frames = Arrays.asList(input.split("\\|"));
```

Comment:
Code that's no longer in use can be deleted.

Reply:
Addressed.